### DISCUSSION TITLE
Can I Make GitHub Pages Private?

### DISCUSSION THREAD

**Discussion by RebelAngel98** [community/community#69715] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I'm going to try to summarize this in the most easiest way. 

I need to make a place where it's possible to be able to put in a specific phrase/number that'll then pull up a 'Menu' that shows all of their past comments. Each comment is put into either Stage Gate 1- Stage Gate 5, depending on which one the user wants. This is all then to be transferred to an excel document, that is forward to a PowerBI Query to be displayed.

Now, the most tricky part is, each project/comment is sensitive information, so I can't exactly do a public GitHub Pages that's to be shown throughout the web. Is there a privacy setting for GitHub Pages that prevents any of that mishap? Or maybe a different suggestion? ALSO - If you have any advice on how to do the project, that is also GREATLY appreciated!

**Comment by ebubekirgungor** [comment]
Hello, if you use Github Pages, it means you only use front-end. If you have sensitive informations in your project, you must use a back-end for your project because in front-end your informations somehow accessible from browser. Some free deploy services compatible with github:
Vercel
Heroku
